source/tutorials/tutorials.rst

```diff
@@ -7,6 +7,18 @@ This section provides tutorials on getting started with NanoVer.
 We assume familiarity with setting up simulations in whichever 
 framework you're running with NanoVer.
 
+The tutorials use `Jupyter notebooks <https://jupyter.org>`_, `NGLView <https://github.com/nglviewer/nglview>`_ for visualising trajectories, and while not strictly necessary, assumes you have the `NanoVer IMD VR <https://gitlab.com/intangiblerealities/nanover-applications/nanover-imd>`_ application installed. These can all be installed with conda:
+
+.. code-block:: bash
+
+   conda activate <nanover_conda_environment_name>
```

Comment:
```suggestion
   # Assuming NanoVer is installed in a conda environment named "nanover",
   # change the name otherwise.
   conda activate nanover
```

In the installation doc (the nanover-protocol README), we call the environment `nanover`. This should allow users just to copy-paste the snippet.

Reply:
That's a great point! This looks like a good change to me, I'll pop that in